backend/src/impl/auth.py

```diff
@@ -48,6 +48,7 @@ def check_BearerAuth(token: str):
 
 class User:
     # keys for fields in self._info
+    _SUB_KEY = "sub"
```

Comment:
I think we need to use "id" in this class and any other places that are not related to Cognito.